docs/initialization-steps-and-the-related-tracer.org

```diff
@@ -5,6 +5,9 @@
 #+LATEX_HEADER: \usepackage[margin=0.9in]{geometry}
 #+OPTIONS: toc:nil
 
+FIXME: this contains way too many code details and it is likely to bitrot. Maybe
```

Comment:
I would move whatever we can extract from this file into the source code and then delete it.